### DISCUSSION TITLE
Cannot Connect to the Codespace

### DISCUSSION THREAD

**Discussion by wraith-tr** [community/community#43059] [Codespaces]
### Select Topic Area

Bug

### Body

I have a project that I am actively developing right now. I was using it over the web. As usual, when I tried to log in today, I started getting the 'failed to connect to codespace' error. When I looked at the console, I saw an error called 'Authentication Fatal Error'. I then deleted all local storage, including cookies and local storage, and tried to log in again. But now I get the error that there is 'no valid authentication key' or 'authentication failed' . Even when I log out of my account and log back in, I still get the same error.

**Comment by erikaellr** [comment]
Hi @wraith-tr, we're thrilled you're using Codespaces, but sorry to hear you were experiencing problems. At the time that you posted, we were actually having an incident that caused failures to connect to codespaces from the web editor. That incident has since been resolved, so I hope that you're now able to connect to any codespaces you're trying to use. If not, please [open a ticket with Support](https://support.github.com/contact).
